### ISSUE TITLE
Error: Cannot find module 'sprintf-js'

### ISSUE THREAD

**Issue by rboorgapally** [joost-de-vries/play-angular2-typescript#4]
Hi,

When I follow the instructions I get the following error:

```
[info] Loading project definition from /home/blr/telugunewsfeed/play-angular2-typescript/project
[info] Updating {file:/home/blr/telugunewsfeed/play-angular2-typescript/project/}play-angular2-typescript-build...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Set current project to play-angular2-typescript (in build file:/home/blr/telugunewsfeed/play-angular2-typescript/)
[info] Updating {file:/home/blr/telugunewsfeed/play-angular2-typescript/}root...
[info] Done updating.

--- (Running the application, auto-reloading is enabled) ---

[info] p.c.s.NettyServer - Listening for HTTP on /0:0:0:0:0:0:0:0:9000

(Server started, use Ctrl+D to stop and go back to the console...)

[info] Compiling 8 Scala sources and 1 Java source to /home/blr/telugunewsfeed/play-angular2-typescript/target/scala-2.11/classes...
[success] Compiled in 4s
[info] Typescript compiling on 3 source(s)
[info] Typescript linting on 3 source(s)
[error]
[error] module.js:340
[error]     throw err;
[error]           ^
[error] Error: Cannot find module 'sprintf-js'
[error]     at Function.Module._resolveFilename (module.js:338:15)
[error]     at Function.Module._load (module.js:280:25)
[error]     at Module.require (module.js:364:17)
[error]     at require (module.js:380:17)
[error]     at Object.<anonymous> (/home/blr/telugunewsfeed/play-angular2-typescript/target/web/node-modules/main/webjars/codelyzer/componentClassSuffixRule.js:8:20)
[error]     at Module._compile (module.js:456:26)
[error]     at Object.Module._extensions..js (module.js:474:10)
[error]     at Module.load (module.js:356:32)
[error]     at Function.Module._load (module.js:312:12)
[error]     at Module.require (module.js:364:17)
[info]
com.typesafe.sbt.jse.SbtJsTask$JsTaskFailure:
module.js:340
    throw err;
          ^
Error: Cannot find module 'sprintf-js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/blr/telugunewsfeed/play-angular2-typescript/target/web/node-modules/main/webjars/codelyzer/componentClassSuffixRule.js:8:20)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)

```

**Comment by rboorgapally** [comment]
If I run npm install sprint-js in the project folder, the error goes away.

**Comment by joost-de-vries** [comment]
Yeah, you're right. It's a mistake I made in the NG2 linting.
I've fixed it in sbt-tslint 0.9.7. If you do a pull it will work without having to install npm modules.
Tx for pointing this out.

**Comment by sanatzzz** [comment]
Hi @rboorgapally  I have installed appium using installer file and i am getting this error. How can i fix it in this case? The error seems to be random. Same installer file has been installed in different machines. Every now and then this errors pops up on one of the machines whereas for others it is perfect.
